packages/generator/gulpfiles/downloadRamlsTask.ts

```diff
@@ -93,13 +91,30 @@ function downloadRamlFromExchange(): Promise<void> {
   });
 }
 
-gulp.task("cleanInputDir", async () => {
-  return fs.removeSync(path.join(config.inputDir));
+gulp.task("diffRamlFiles", async () => {
+  const result = await diffNewAndArchivedRamlFiles(
+    config.apiBackupDir,
+    config.inputDir,
+    path.join(config.apiBackupDir, config.apiConfigFile),
+    path.join(config.inputDir, config.apiConfigFile)
+  );
+  return fs.writeFile(config.diffFile, JSON.stringify(result));
+});
+
+gulp.task("cleanup", async () => {
+  fs.removeSync(path.join(config.apiBackupDir));
+  fs.removeSync(config.diffFile);
+  return fs.moveSync(
+    path.join(config.inputDir),
+    path.join(config.apiBackupDir)
+  );
```

Comment:
```suggestion
  fs.removeSync(config.apiBackupDir);
  fs.removeSync(config.diffFile);
  return fs.moveSync(config.inputDir, config.apiBackupDir);
```
Wrapping a single argument in `join` is unnecessary.